libs/openFrameworks/app/ofAppRunner.cpp

```diff
@@ -193,6 +193,10 @@ void ofExitCallback(){
 		timeEndPeriod(1);
 	#endif
 
+	//------------------------
+	// try to close font libraries
```

Comment:
Nah, I just copied and pasted what other comments said :D Feel free to change any of my pull requests after their merge!